### PR TITLE
add parsnip v1.0.2 blog post

### PR DESCRIPTION
**Pull request by simonpcouch**
A draft of the blog post for parsnip v1.0.2. Specifically highlights changes to improvements to errors and warnings, and does so a bit more in the weeds than previous parsnip release posts.

cc @topepo—feel free to push additions directly to this branch (if you have permissions to do so)!

[edit: review request assigned🏄‍♀️]

### CODE REVIEW

File: content/blog/parsnip-checking/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+#> ! 'beep bop boop' is not a known mode for model `bag_tree()`.
+```
+
+We hope these changes improve folks' experience when modeling with parsnip in the future!
+
+## Other bits and bobs
+
+<!-- This post has highlighted upcoming improvements to model specification checking in parsnip. For those who'd like to learn more, I've written a [companion article](https://simonpcouch.com/blog) on my blog that delves further into the tooling we use to check model specifications. -->
+
+parsnip v1.0.2 includes a number of other changes that you can read about [here](https://github.com/tidymodels/parsnip/blob/main/NEWS.md).
```
Comment (hfrick) [inline content/blog/parsnip-checking/index.Rmd:357]: I think this is getting lost a bit here. I would keep this as the usual thank you section and move that link up to the first section. Something like "this post focuses on xyz but you can see the full [release notes](link to the release notes of specifically this version, once the pkgdown site has been updated) for all other changes".

**Review by hfrick — APPROVED** [review]
Really nice example and a great walk-through! I left some suggestions, pick the ones you like!

File: content/blog/parsnip-checking/index.Rmd (hunk 1 above)
Comment (simonpcouch) [reply content/blog/parsnip-checking/index.Rmd]: Addressed in [e5f39c4](https://github.com/tidyverse/tidyverse.org/pull/597/commits/e5f39c492bc4d37322e94efeaef522b4f34f2c3e). :)

File: content/blog/parsnip-checking/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [x] `usethis::use_tidy_thanks()`
+-->
+
+We're stoked to announce the upcoming release of [parsnip](https://parsnip.tidymodels.org/) v1.0.2 on CRAN! parsnip provides a tidy, unified interface to models that can be used to try a range of models without getting bogged down in the syntactical minutiae of the underlying packages. This release includes improvements to errors and warnings that proliferate throughout the tidymodels ecosystem. These changes are meant to better anticipate common mistakes and nudge users informatively when defining model specifications.
+
+First, though, we'll start off with a higher-level overview of parsnip and its foundational role in the tidymodels collection of packages.
```
[the remark this answers is not in the document]
Comment (simonpcouch) [reply content/blog/parsnip-checking/index.Rmd]: Addressed in [e5f39c4](https://github.com/tidyverse/tidyverse.org/pull/597/commits/e5f39c492bc4d37322e94efeaef522b4f34f2c3e). :)

**Comment by simonpcouch** [comment]
Thank you, @hfrick!🏄 I set the date for this coming Wednesday, but we can of course bump as needed.

File: content/blog/parsnip-checking/index.md [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+* [x] Pick category and tags (see existing with [`hugodown::tidy_show_meta()`](https://rdrr.io/pkg/hugodown/man/use_tidy_post.html))
+* [x] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] [`hugodown::use_tidy_thumbnails()`](https://rdrr.io/pkg/hugodown/man/use_tidy_post.html)
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [x] [`usethis::use_tidy_thanks()`](https://usethis.r-lib.org/reference/use_tidy_thanks.html)
+-->
+
+We're stoked to announce the upcoming release of [parsnip](https://parsnip.tidymodels.org/) v1.0.2 on CRAN! parsnip provides a tidy, unified interface to various statistical and machine learning models. This release includes improvements to errors and warnings that proliferate throughout the tidymodels ecosystem. These changes are meant to better anticipate common mistakes and nudge users informatively when defining model specifications. parsnip v1.0.2 includes a number of other changes that you can read about in the [release notes](https://github.com/tidymodels/parsnip/blob/main/NEWS.md).
```
Comment (topepo) [inline content/blog/parsnip-checking/index.md:37]:
```suggestion
We're stoked to announce the new release of [parsnip](https://parsnip.tidymodels.org/) v1.0.2 on CRAN! parsnip provides a tidy, unified interface to various statistical and machine learning models. This release includes improvements to errors and warnings that proliferate throughout the tidymodels ecosystem. These changes are meant to better anticipate common mistakes and nudge users informatively when defining model specifications. parsnip v1.0.2 includes a number of other changes that you can read about in the [release notes](https://github.com/tidymodels/parsnip/blob/main/NEWS.md).
```

File: content/blog/parsnip-checking/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,357 @@
...
+#> Bagged Decision Tree Model Specification (unknown mode)
+#> 
+#> Main Arguments:
+#>   cost_complexity = 0
+#>   min_n = 2
+#> 
+#> Computational engine: rpart
+```
+
+If we specify the mode as a regression model, though, the specification is no longer well-supported given the extensions we’ve loaded. parsnip will again prompt us to load the correct package:
```
Comment (topepo) [inline content/blog/parsnip-checking/index.Rmd:207]: > is no longer well-supported given the extensions we’ve loaded

Can you reword this?

**Comment by simonpcouch** [comment]
Comments resolved in 4a2472d!

**Pull request merged**